src/extension/alterschema/linter/then_false.h

```diff
@@ -0,0 +1,39 @@
+class ThenFalse final : public SchemaTransformRule {
+public:
+  ThenFalse()
+      : SchemaTransformRule{
+            "then_false",
+            "`if: S, then: false` is logically equivalent to `not: S`"} {};
```

Comment:
```suggestion
            "`Setting the `then` keyword to the false schema is a convoluted way of negating the `if` subschema, which you can more cleanly represent using the `not` keyword"} {};
```

Just to make it a lot more clear to users